matrix_benchmarking/download_lts.py

```diff
@@ -61,19 +60,44 @@ def run():
             ssl_show_warn=False,
         )
 
-        download(client, kwargs.get("opensearch_index"), kwargs.get("filters"), pathlib.Path(kwargs.get("results_dirname")))
+        download(
+            client,
+            kwargs.get("opensearch_index"),
+            kwargs.get("filters"),
+            pathlib.Path(kwargs.get("results_dirname")),
+            kwargs.get("max_records")
+        )
 
     return cli_args.TaskRunner(run)
 
 
-def download(client, opensearch_index, filters, results_dirname):
+def download(client, opensearch_index, filters, results_dirname, max_records):
     lts_dir_anchor = results_dirname / LTS_ANCHOR_NAME
     if lts_dir_anchor.exists():
         logging.critical(f"{lts_dir_anchor} already exists, cannot continue.")
         return 1
 
     logging.info(f"Querying OpenSearch {opensearch_index} ...")
-    search = client.search(index=opensearch_index)
+
+    query = {
+        "size": max_records
+    }
+
+    # Restrict the results to specific settings
+    if len(filters) > 0:
```

Comment:
simpler/more pythonic this way:
```
if filters:
```
(not sure why github doesn't let me put it as a suggestion 🤔 )

Reply:
Gotcha, I thought the cli arg parsing would make the default filters `{}`. Will fix.

Reply:
`if filters:` will be false with any of  `""`, `[]`, `{}`, and true if the string/list/dict has some content

Reply:
Oh that's cool! I didn't know those were considered `False`